kernel/kernel-config-4.9.x

```diff
@@ -35,6 +35,7 @@ CONFIG_ZONE_DMA32=y
 CONFIG_AUDIT_ARCH=y
 CONFIG_ARCH_SUPPORTS_OPTIMIZED_INLINING=y
 CONFIG_ARCH_SUPPORTS_DEBUG_PAGEALLOC=y
+CONFIG_HAVE_INTEL_TXT=y
```

Comment:
@amshinde was this needed?

Reply:
@mcastelino That seemed a little off to me as well, but that seems to be a side-effect of  enabling VFIO and IOMMU configs. None of the help text for the configs I enabled, mentions about this CONFIG getting enabled. But I can confirm based on some online searches that this does get set when IOMMU is enabled. Perrhaps because of this https://cateee.net/lkddb/web-lkddb/INTEL_IOMMU_DEFAULT_ON.html

Reply:
This is a boolean flag to enable TXT below, and there is disabled

Reply:
@miguelinux @amshinde as we do not have IOMMU support within the guest, will VFIO work? Can you check if you can actually use VFIO within the CC VM. vfio actually support "enable_unsafe_noiommu_mode" which allows it to be used without an IOMMU. 

So can you let us know what you see within the VM before we merge the same.

I assume the TXT came in here due to DMAR requirements. And we do not support that either within the VM.